Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -222,15 +222,19 @@ def unisolate_machine(self, machine_id, comment):
         }
         return self.ms_client.http_request(method='POST', url_suffix=cmd_url, json_data=json_data)
 
-    def get_machines(self, filter_req):
+    def get_machines(self, filter_req, next_link, page_size):
         """Retrieves a collection of Machines that have communicated with Microsoft Defender ATP cloud on the last 30 days.
 
         Returns:
             dict. Machine's info
         """
-        cmd_url = '/machines'
+        cmd_url = f'/machines?$top={int(page_size)}'
```

Comment:
better to pass this in the params as done below
also the `int` casting is probably not required